build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/tscTask.ts

```diff
@@ -484,32 +508,41 @@ export class TscMultiTask extends LeafWithDoneFileTask {
 	protected async getDoneFileContent(): Promise<string | undefined> {
 		const command = this.command;
 
-		/**
-		 * A list of files that should be considered part of the cache input, if they exist
-		 */
-		const commonFiles = [
-			"tsconfig.json",
-			"src/test/tsconfig.json",
-			"tsc-multi.json",
-			"tsc-multi.test.json",
-		]
-			.map((file) => path.resolve(this.package.directory, file))
-			.filter((file) => existsSync(file));
-
 		try {
-			// The path to the tsbuildinfo file differs based on if it's a CJS vs. ESM build. Use the presence of "esnext" in
-			// the command string to determine which file to use.
-			const tsbuildinfoPath = this.getPackageFileFullPath(
-				command.includes("tsc-multi.esm.json")
-					? "tsconfig.mjs.tsbuildinfo"
-					: "tsconfig.cjs.tsbuildinfo",
+			const commandArgs = command.split(/\s+/);
```

Comment:
Do you care about quoted arguments with spaces?  (e.g., "--config 'my/obnoxious/path with spaces'")

I assume FluidBuild either generally doesn't handle these, or there would be a utility function you could reuse here.